set.mm

```diff
@@ -655637,6 +655679,43 @@ not even needed (it can be any class).  (Contributed by Glauco
       $.
   $}
 
+  ${
+    fnssresd.1 $e |- ( ph -> F Fn A ) $.
+    fnssresd.2 $e |- ( ph -> B C_ A ) $.
+    $( Restriction of a function with a subclass of its domain.  (Contributed
```

Comment:
with -> to

Reply:
Change as suggested, in the latest commit, but please notice that Norman wrote such comments with "with" instead of "to", see ~ fnssresb , for example.

Reply:
after checking wikipedia, planetmath, encyclopedia of math, I confirm "to"